BreadcrumbsItem.astro

```diff
@@ -25,4 +25,13 @@ const { href = '#', label = 'Breadcrumb', currentPage = false } = Astro.props
   .breadcrumbs__item:last-child::after {
     content: '';
   }
+
+  .breadcrumbs__item a[aria-current]:not([aria-current="false"]) {
```

Comment:
Hey@roybarber ... looks good... any thoughts on using `[aria-current="page"]` in the style instead? Just because we are explicitly setting "page" in the component?

`.breadcrumbs__item a[aria-current="page"]` rather than having to check for "false" ?